test/torchaudio_unittest/transforms/batch_consistency_test.py

```diff
@@ -43,7 +43,7 @@ def test_batch_Resample(self):
     def test_batch_MelScale(self):
         specgram = torch.randn(3, 2, 201, 256)
 
-        atol = 1e-6 if os.name == "nt" else 1e-8
+        atol = 1e-4 if os.name == "nt" else 1e-6
```

Comment:
can you revert back this change? these values should be fine for the original melscale tests

Reply:
Yes of course. When a ran the test locally, MelScale didn't satisfy these values, so I played a little bit with the values and forgot to revert the changes. You can test it on your own.

Reply:
![Screenshot_20221103_103546](https://user-images.githubusercontent.com/7811453/199688402-6b68060c-1334-4103-9d6e-968f56619d75.png)

This is the output of batch_consistency_test with the original tolerance values.

Reply:
Is there any feedback about this?

Reply:
Hey, yea it is also erroring out for me with those values locally, but seems to run okay on our GitHub CircleCI tests -- I think we can leave as is for now, and I can also test out to see if bark scale is passing on CircleCI tests with smaller tolerances after this PR is merged.